drivers/lvutil.py

```diff
@@ -138,6 +138,34 @@ def __exit__(self, exc_type, value, traceback):
             self.lock.release()
 
 
+LVM_RETRY_ERRORS = [
+    "Incorrect checksum in metadata area header"
```

Comment:
Isn't this an indication that we're missing a lock somewhere, or not flushing updates properly?
Retrying might fix the issue on read-only operations, but don't we also have an issue where 2 writers could race and end up in a lost update if the 2nd writer started out with slightly older information and overwrote the changes made by the other writer?

Reply:
Absolutely we're missing a lock, we're missing a lock because we abuse LVM and use it from multiple hosts without a corosync cluster. This is an immense pile of technical debt that we can only patch up and bandage and hope and pray that thing hang together.